### PR TITLE
update_reader_plugin

### PR DESCRIPTION
**Pull request by CharmisG**
Plugin upgration for android reader-sdk

### CODE REVIEW

File: example/pubspec.yaml [hunk 1]
```diff
@@ -10,19 +10,19 @@ description: Demonstrates how to use the square_reader_sdk plugin.
...
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.14.0 <3.0.0'
 
 dependencies:
   flutter:
     sdk: flutter
 
-  permission_handler: ^8.0.0+2
+  permission_handler: any
```
Comment (hukid) [inline example/pubspec.yaml:19]: Is that a good idea to pick any version of `permission_handler`?

**Comment by hukid** [comment]
Is this change for upgrading the example file or the plugin itself?

File: example/pubspec.yaml (hunk 1 above)
Comment (CharmisG) [reply example/pubspec.yaml:19]: yes we don't need to give specific version. it will take latest version automatically.

**Comment by CharmisG** [comment]
> @CharmisG Is this change for upgrading the example file or the plugin itself?

I have upgraded for parent plugin also.

File: example/pubspec.yaml (hunk 1 above)
Comment (hukid) [reply example/pubspec.yaml:19]: can the latest version break the app unexpectedly?

**Review by hukid — APPROVED** [review]

File: example/pubspec.yaml (hunk 1 above)
Comment (CharmisG) [reply example/pubspec.yaml]: I changed version any to specific version.

**Pull request merged**